src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/monitoring/main.tf

```diff
@@ -176,6 +181,9 @@ resource "helm_release" "prometheus-grafana" {
           "${var.node-group.key}" = var.node-group.value
         }
 
+        # Avoid using the default password, as that's a security risk
+        adminPassword : random_password.grafana_admin_password.result
```

Comment:
I don't see this stored anywhere. Is the assumption that we won't ever need it?

Reply:
It is stored as a secret, created and managed by the grafana helm chart.